### PR TITLE
adding message to the db become a function, plus logging method is changed a bti

### PR DESCRIPTION
**Pull request by tk0nik**
**about changes in logging:**
now **all** the messages are posted into console, but upper than DEBUG ones (ERROR, INFO etc) - into Telegram channel. 

`client` was not nesessary parameter for the `debug` function. In current verwion of freegan. In other places it's used just like a global variable. 

Plus minor changes:

```from logging import DEBUG, INFO, WARNING, ERROR, CRITICAL``` 
is the same as 
```
DEBUG = logging.DEBUG
INFO = logging.INFO
WARNING = logging.WARNING
ERROR = logging.ERROR
CRITICAL = logging.CRITICAL
```
`LEVEL_NAMES` dict was useless.

**about post_message_to_db_archive method**
has to be tested with API errors later

### CODE REVIEW

**Comment by lejafo** [comment]
is it just me or does this branch need to be merged with main?

**Review by lejafo — CHANGES_REQUESTED** [review]
merge branch from main

**Comment by tk0nik** [comment]
> merge branch from main

it was already done yesterday. Now I'm getting `Nothing to merge, logging_task is already up to date` message when I try

**Comment by lejafo** [comment]
> > merge branch from main
> 
> it has already been done yesterday. Now I'm getting `Nothing to merge, logging_task is already up to date` message when I try

but i see that at least the lines between 65 and 77 shouldn't be there

```
intersection_group_1 = lemmas.intersection(keyword_group_1)
...
```

**Comment by tk0nik** [comment]
> > > merge branch from main
> > 
> > 
> > it has already been done yesterday. Now I'm getting `Nothing to merge, logging_task is already up to date` message when I try
> 
> but i see that at least the lines between 65 and 77 shouldn't be there
> 
> ```
> intersection_group_1 = lemmas.intersection(keyword_group_1)
> ...
> ```

I've just cut them out and everything seems ok to me now. Don't know why that happened

**Comment by Dogadaeva** [comment]
Is post_message_to_db_archive function not in use now?

File: main.py [hunk 1]
```diff
@@ -10,22 +10,12 @@
 from datetime import datetime
 from lemmatization import lemmatize
 import requests
+from logging import DEBUG, INFO, WARNING, ERROR, CRITICAL
```
Comment (MSenso) [inline main.py:13]: Great

**Review by MSenso — APPROVED** [review]

**Review by lejafo — APPROVED** [review]

**Comment by tk0nik** [comment]
> Is post_message_to_db_archive function not in use now?

we can switch to 'False' the `on` parameter 
Actually could be worthy to create a constant for it or maybe put it into config

**Comment by tk0nik** [comment]
> > Is post_message_to_db_archive function not in use now?
> 
> it has to be inactive right now
> 
> we can switch to 'False' the `on` parameter Actually could be worthy to create a constant for it or maybe put it into config

Damn, I see what's it about. Changes when this staff is called aren't pulled *facepalm*

**Review by Dogadaeva — APPROVED** [review]

File: main.py [hunk 2]
```diff
@@ -209,15 +211,15 @@ async def run_client():
     async with client:
         await check(client)
         try:
-            await debug(client, "Freegan has started", INFO)
+            # await debug("Freegan has started", INFO)
```
Comment (lejafo) [inline main.py:214]: why is the start log message is commented out?

File: main.py [hunk 3]
```diff
@@ -36,6 +26,7 @@
 api_id = int(os.getenv('TELEGRAM_API_ID', 0))
 api_hash = str(os.getenv('TELEGRAM_API_HASH'))
 chat_send_to = str(os.getenv('TELEGRAM_CHAT_SEND_TO'))
+messages_collecting_is_on = str(os.getenv('MESSAHES_COLLECTING_IS_ON'))
```
Comment (lejafo) [inline main.py:29]: spelling mistake

Comment (lejafo) [reply main.py:29]: you need also to add MESSAGES_COLLECTING_IS_ON to the env-example file

**Review by MSenso — CHANGES_REQUESTED** [review]
Fix spelling mistakes, add new env variable into env.example, remove old codelines that are commented now. Besides that, it looks good

File: main.py (hunk 3 above)
Comment (tk0nik) [reply main.py]: done I guess

File: main.py (hunk 2 above)
Comment (tk0nik) [reply main.py]: Oh, it was useless for me while testing and I'm not sure it has to be in the final version. But for now I've resurrected it :)

**Review by MSenso — APPROVED** [review]

**Pull request merged**